Cesium3DTilesSelection/src/ImplicitQuadtreeLoader.cpp

```diff
@@ -115,9 +129,11 @@ CesiumAsync::Future<TileLoadResult> requestTileContent(
     const std::vector<CesiumAsync::IAssetAccessor::THeader>& requestHeaders,
     CesiumGltf::Ktx2TranscodeTargets ktx2TranscodeTargets,
     bool applyTextureTransform,
-    const glm::dmat4& tileTransform) {
+    const glm::dmat4& tileTransform,
+    const CesiumGeospatial::Ellipsoid& ellipsoid) {
   return pAssetAccessor->get(asyncSystem, tileUrl, requestHeaders)
-      .thenInWorkerThread([pLogger,
+      .thenInWorkerThread([&ellipsoid,
```

Comment:
This capture of the ellipsoid by reference looks unsafe. What guarantees do we have that the ellipsoid passed to requestTileContent won't be destroyed before the request completes and the worker thread lambda runs?